### PR TITLE
[feature](function) support ip function ipv6_string_to_num(_or_default, _or_null), inet6_aton

### PR DESCRIPTION
**Pull request by superdiaodiao**
## Proposed changes
Issue Number: #21489

Issue Number: #19624

**add functions**:
ipv6_string_to_num(alias inet6_aton), 
ipv6_string_to_num_or_default, 
ipv6_string_to_num_or_null

**Doris output**:

image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morrySnow — DISMISSED** [review]

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run beut

File: docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/ipv6-string-to-num-or-default.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+{
+"title": "IPV6_STRING_TO_NUM_OR_DEFAULT",
+"language": "en"
+}
```
Comment (HappenLee) [inline docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/ipv6-string-to-num-or-default.md:5]: not en，please change

File: be/src/vec/functions/function_ip.h [hunk 2]
```diff
@@ -348,4 +348,254 @@ class FunctionIPv6NumToString : public IFunction {
...
+
+    static FunctionPtr create() {
+        return std::make_shared<FunctionIPv6StringToNum<exception_mode>>();
+    }
+
+    String get_name() const override { return name; }
+
+    size_t get_number_of_arguments() const override { return 1; }
+
+    bool use_default_implementation_for_nulls() const override { return false; }
```
Comment (HappenLee) [inline be/src/vec/functions/function_ip.h:557]: why here use false ？

Comment (superdiaodiao) [reply be/src/vec/functions/function_ip.h:557]: to keep the behaviors like Clickhouse

File: docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/ipv6-string-to-num-or-default.md (hunk 1 above)
Comment (superdiaodiao) [reply docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/ipv6-string-to-num-or-default.md:5]: ok, I will fix

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run feut

**Comment by superdiaodiao** [comment]
run clickbench

**Comment by superdiaodiao** [comment]
run clickbench

**Comment by superdiaodiao** [comment]
run buildall

**Comment by superdiaodiao** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by amorynan — APPROVED** [review]
LGTM

**Pull request merged**